testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -199,11 +199,11 @@ QUnit.test("done button is located before cancel button in win10", function(asse
         instance = $popup.dxPopup("instance"),
         $popupBottom = instance.$content().parent().find(".dx-popup-bottom");
 
-    assert.equal($popupBottom.text(), "DoneCancel", "buttons order is correct");
+    assert.equal($popupBottom.text(), "OKCancel", "buttons order is correct");
```

Comment:
We have internal shortcuts for toolbar button - `['done', 'clear', 'cancel']`.

What do you think about rename the 'done' shortcut to 'ok'?
You add a "done" button and check that its text is "OK". It's confusing a little.